test/components/DayPickerRangeController_spec.jsx

```diff
@@ -12,7 +12,7 @@ import isInclusivelyAfterDay from '../../src/utils/isInclusivelyAfterDay';
 
 import { START_DATE, END_DATE } from '../../constants';
 
-const today = moment().startOf('day');
+const today = moment().startOf('day').hours(12);
```

Comment:
That 'today' is in three other specs. For consistency I think it would be better to bring them all in line with this. I can comment on each.